### PR TITLE
Add additional issue filters and prototype daily digest/email app

### PR DESCRIPTION
**Pull request by chadpowers**
@dpwspoon

### CODE REVIEW

File: src/main/java/org/kaazing/github/Action.java [hunk 1]
```diff
@@ -194,4 +211,168 @@ public String getDescription() {
         }
     };
 
+    public static Action LIST_ISSUES_CLOSED_WITHIN_DAY = new Action() {
+
+        @Override
```
Comment (dpwspoon) [inline src/main/java/org/kaazing/github/Action.java:222]: Minor: There are formatting issues with whitespace here, I can show you how we do code formatting at kaazing if you stop by sometime

File: src/main/java/org/kaazing/github/Action.java [hunk 2]
```diff
@@ -194,4 +211,168 @@ public String getDescription() {
...
+
+    };
+    
+    public static Action LIST_ISSUES_UPDATED_WITHIN_DAY = new Action() {
+
+        @Override
+        public void accept(GHRepository t) {
+        	List<GHIssue> closed = null;
+        	try {
+        		closed = t.getIssues(GHIssueState.OPEN);
```
Comment (dpwspoon) [inline src/main/java/org/kaazing/github/Action.java]: Minor: You can static import these states I believe to make the code more readable

File: src/main/java/org/kaazing/github/apps/DailyEmail.java [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+    		}
+    	}
+    	htmlText += "\t\t</ul>\n\t</li>\n</ul>\n";
+    	
+    	if (addedSomething)
+    		return userText;
+    	return "";
+    }
+    
+    // get/save text for no assignee
```
Comment (dpwspoon) [inline src/main/java/org/kaazing/github/apps/DailyEmail.java]: Probably better as a javadoc like comment, ditto for others

File: README.md [hunk 4]
```diff
@@ -47,12 +56,47 @@ byName(name, name2 etc)	        gets a repo matching the name
...
+UPDATED_WITHIN_DAYS(numberDays)		matches issues updated within argument days
+HAS_LABEL(label)					matches issues containing argument label
+HAS_ASSIGNEE(login)					login of user, null argument returns issues with no assignee
+BY_MILESTONE(milestone)				filters by milestone
+BY_NAME								gets issue matching the name
 ```
+
+# Apps
+### DailyEmail.java
+Compiles and outputs both plain text (command line) and html (out_email.html) of users and key issues.
```
Comment (dpwspoon) [inline README.md:92]: Can you add instructions on how to run it from the command line, (i.e. after it has been compiled)

Comment (chadpowers) [inline README.md:92]: Added

**Comment by dpwspoon** [comment]
Thanks @chadpowers

**Pull request merged**